packages/web/hooks/limit-orders/use-place-limit.ts

```diff
@@ -1,6 +1,7 @@
 import { CoinPretty, Dec, Int, PricePretty } from "@keplr-wallet/unit";
 import { priceToTick } from "@osmosis-labs/math";
 import { DEFAULT_VS_CURRENCY } from "@osmosis-labs/server";
+import { cosmwasmMsgOpts } from "@osmosis-labs/stores";
```

Comment:
Generally we consider stores package to be deprecated. Were trying to move msg/tx related logic to tx package. I'm moving them in `jonator/fixes` branch

Reply:
Just merged to stage

Reply:
Just merged this move to stage, pls update imports (and anywhere else you import this from)

Can just change it from /stores to /tx